webview_flutter/step_12/test/widget_test.dart

```diff
@@ -0,0 +1,18 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:step_12/main.dart';
+
+void main() {
+  testWidgets('Do nothing test', (tester) async {
+    // Build our app and trigger a frame.
+    await tester.pumpWidget(const MaterialApp(home: WebViewApp()));
```

Comment:
If you are not goin to write any test, smoke this at least

```suggestion
    await tester.pumpWidget(const MaterialApp(home: WebViewApp()));
    
    // smoke test
    expect(find.byType(WebViewApp), findsOneWidget);
```